### ISSUE TITLE
module 'autokeras' has no attribute 'auto_model' - bug report

### ISSUE THREAD

**Issue by gkourogiorgas** [keras-team/autokeras#775]
I tried to run `autokeras.auto_ml.AutoModel `in Google's Colab (python 3.6.8)
`!pip install numpy==1.15.4 autokeras`

Then
```
import autokeras

ak=autokeras.auto_model.AutoModel(predictors, ['highfwd'], name='auto_model', max_trials=100, directory=None, seed=None)
ak.fit(x=X_train, y=y_train, validation_split=0.15)
```

### Bug Description

I get the error:
```module 'autokeras' has no attribute 'auto_model'```

### Reproducing Steps
  * !pip install numpy==1.15.4 autokeras
  * run a simple automodel

### Setup Details
Include the details about the versions of:
 - OS type and version: Google Colab
 - Python: 3.6.8
 - autokeras: 0.4.0
 - scikit-learn: 0.21.3
 - numpy: 1.15.4
 - keras: 2.2.5 
 - scipy: 1.2.0 
 - tensorflow: 1.13.1 
 - pytorch:

**Comment by dsandeep0138** [comment]
auto_model is introduced in autokeras-1.0. Try this version using the following commands and please see if it works for you...

```
pip install git+git://github.com/keras-team/autokeras@master#egg=autokeras
pip install git+git://github.com/keras-team/keras-tuner@master#egg=keras-tuner
import autokeras
```

We can check the same using the following commands:

autokeras 0.4 code
```
import autokeras
print(dir(autokeras))
```
['CnnModule', 'ImageClassifier', 'ImageRegressor', 'MlpModule', 'PortableImageSupervised', 'TextClassifier', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', 'backend', 'bayesian', 'constant', 'custom_queue', 'image', 'net_module', 'net_transformer', 'nn', 'preprocessor', 'search', 'supervised', 'text', 'utils']

autokeras 1.0 code
```
import autokeras
print(dir(autokeras))
```
['AutoModel', 'ClassificationHead', 'Constant', 'ConvBlock', 'DenseBlock', 'EmbeddingBlock', 'FeatureEngineering', 'GraphAutoModel', 'ImageAugmentation', 'ImageBlock', 'ImageClassifier', 'ImageInput', 'ImageRegressor', 'Input', 'LightGBMBlock', 'Merge', 'Normalization', 'RNNBlock', 'RegressionHead', 'ResNetBlock', 'SpatialReduction', 'StructuredDataBlock', 'StructuredDataClassifier', 'StructuredDataInput', 'StructuredDataRegressor', 'TemporalReduction', 'TextBlock', 'TextClassifier', 'TextInput', 'TextRegressor', 'TextToIntSequence', 'TextToNgramVector', 'XceptionBlock', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', 'auto_model', 'const', 'encoder', 'hypermodel', 'meta_model', 'task', 'tuner', 'utils']
